var/spack/repos/builtin/packages/wxwidgets/package.py

```diff
@@ -52,6 +53,8 @@ def configure_args(self):
 
         if self.spec.satisfies("+opengl"):
             options.append("--with-opengl")
+        if not self.spec.satisfies("+gui"):
```

Comment:
For future reference, you could use the `enable_or_disable` helper (https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#helper-functions).